memory.tex

```diff
@@ -1844,19 +1874,26 @@ \subsection{Converting ROM to RAM}
 RAM Memory Map describes which ROM regions are used for each purpose.
 
 A carefully written program that doesn't need the built-in routines at all can
-repurpose the entire 128KB region. It can load in alternate BASIC and kernel
-code. Naturally, this overrides the MEGA65 ROM that the user has installed, and
-so can't take advantage of improvements in newer versions of the ROM.
+repurpose the entire 128KB region. For example, it can load in alternate BASIC
+and kernel code. Naturally, this overrides the MEGA65 ROM that the user has
+installed, and so can't take advantage of improvements in newer versions of
+the ROM.
 
 To toggle the read-only status of banks 2 and 3, invoke the Hypervisor Toggle
-ROM Write-protect system trap, like so:
+ROM Write-protect system trap ({\tt hyppo\_toggle\_rom\_writeprotect}), like so:
 
 \begin{screenoutput}
 LDA #$70
 STA $D640
 NOP
```

Comment:
Better use clv instead of nop. It is non-deterministic whether this command will be executed or not. clv is neutral in this regard, similar to nop, but the latter might have an effect changing the meaning of the following opcode (which may become a compound instruction where nop is used as the prefix opcode).

Reply:
Fixed. Agreed we should use CLV in all hyppo trap examples.